apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -98,22 +99,30 @@ class LoginTypePicker extends Component {
         width: styleConstants['content-width'],
         height: '100px',
         left: 0,
-        bottom: '-65px',
+        bottom: '-51px',
         padding: '0px 20px 20px 20px',
         backgroundColor: '#fff',
         borderRadius: '5px'
       },
+      mediumText: {
+        fontSize: '.75em',
+        color: color.neutral_dark,
+        fontFamily: '"Gotham 5r", sans-serif'
+      },
+      learnHow: {
+        marginTop: '12px'
+      },
       emailPolicyNote: {
-        marginBottom: '20px',
-        paddingTop: '20px',
-        borderTop: '1px solid #000'
+        marginBottom: '31px',
+        paddingTop: '8px',
+        borderTop: `1px solid ${color.neutral_dark}`
       }
     };
 
     return (
       <div style={style.container}>
-        <Heading1>{title}</Heading1>
-        <Heading2>{i18n.addStudentsToSectionInstructionsUpdated()}</Heading2>
+        <Heading3>{title}</Heading3>
```

Comment:
i can't find a definitive answer on what starting heading level inside a modal is best for screenreaders, but i would guess that h2 or h1 would be easier to navigate. would the `Heading3`s in here work as `Heading2` or `Heading1` with a class to adjust the styling?

that said, this modal already has other major accessibility issues, so I wouldn't consider this a blocking comment.

Reply:
Not sure why h3 should be a problem for screenreader. h3 is still a header so I don't see any semantic issues here, correct me if I'm wrong. In my opinion it's better to stick with one type-specs that we agreed and put in [this file](https://staging.code.org/rebrand/rebrand-type-spec#)

Reply:
As an option - we can discuss it with designer, Molly Peredo